pkgs/development/python-modules/outlines-core/default.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  cargo,
+  rustPlatform,
+  rustc,
+  setuptools-rust,
+  setuptools-scm,
+  interegular,
+  jsonschema,
+  datasets,
+  numpy,
+  pytestCheckHook,
+  pydantic,
+  scipy,
+  torch,
+  transformers,
+}:
+
+buildPythonPackage rec {
+  pname = "outlines-core";
+  version = "0.1.17";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "outlines_core";
+    hash = "sha256-CSWhKrHogYxvGJb+79tNTYySLmRhUVBi0b0+YEWbAoE=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    ls -l
+    cat ${./Cargo.lock} > Cargo.lock
+    ls -l
+  '';
+
+  pyproject = true;
+
+  build-system = [
+    cargo
+    rustPlatform.cargoSetupHook
+    rustc
+    setuptools-rust
+    setuptools-scm
+  ];
```

```suggestion
  nativeBuildInputs = [
    cargo
    rustc
    rustPlatform.cargoSetupHook
  ];

  build-system = [
    setuptools
    setuptools-rust
    setuptools-scm
  ];
```
`build-system` should only contain the Python build system

---

Not sure what happened, but it's still here

---

I think something went wrong while rebasing. Should be fixed now.